spec/models/form526_submission_spec.rb

```diff
@@ -168,13 +198,15 @@
           remediated_and_expired,
           remediated_and_rejected,
           happy_path_success,
-          accepted_backup
+          accepted_backup,
+          paranoid_success,
+          success_by_age
         )
       end
     end
 
     describe 'failure_type' do
-      it 'returns anything not explicitly successful or still in process' do
+      it 'returns anything not successful of in process' do
```

Comment:
Should this be '**or** in process'?

Reply:
just reverting it to the original actually